test/org/sagebionetworks/bridge/services/ScheduledActivityServiceDuplicateTest.java

```diff
@@ -45,7 +45,6 @@
  * a standard time and midnight seems to work regardless of when the timestamp is).
  */
 @RunWith(MockitoJUnitRunner.class)
-@Ignore // The current code does not prevent duplicates.
 public class ScheduledActivityServiceDuplicateTest {
```

Comment:
None of these tests seem to test the scenario where the user creates a bunch of events in one timezone, then asks for activities in another timezone. Are we sure this test actually tests what we're testing for?

Also, we test for the presence of tasks we expect. What about the absence of tasks we *don't* expect? iirc, the duplicated tasks showed up because they had different timestamps that were off by an hour (equal to the change in timezone). Should our tests verify that that the task with the wrong timestamp isn't there?